packages/linux/package.mk

```diff
@@ -73,6 +73,7 @@ if [ "$TARGET_KERNEL_ARCH" = "arm64" -a "$TARGET_ARCH" = "arm" ]; then
   PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET gcc-linaro-aarch64-linux-gnu:host"
   PKG_TARGET_PREFIX=$TOOLCHAIN/lib/gcc-linaro-aarch64-linux-gnu/bin/aarch64-linux-gnu-
   HEADERS_ARCH=$TARGET_ARCH
+  PKG_BUILD_PERF="no"
```

Comment:
Apologies for posting after the merge, but while this won't build `perf` it will still pull in the `perf` dependencies which will be built unnecessarily.

Moving the `perf` test after this block and changing the test as follows would work better:
```
if [ "$DEVTOOLS" = "yes" -a "$PKG_BUILD_PERF" != "no"  ] && grep -q ^CONFIG_PERF_EVENTS= $PKG_KERNEL_CFG_FILE ; then
```

Reply:
@MilhouseVH thanks, I will sent a PR to master with your suggestion and drop this commit on next rebase.